deploy/examples/monitoring/rbac.yaml

```diff
@@ -111,3 +112,40 @@ subjects:
     namespace: rook-ceph
 # OLM: END ROLE BINDING
 ---
+# OLM: BEGIN ROLE
+# Allow management of monitoring resources in the exporter
+kind: Role
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: rook-ceph-monitor-exporter
```

Comment:
The exporter pod isn't working with the servicemonitors directly, so we don't need this role.